cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.5.0
+- Remove `--Hint` argument from cli
```

Comment:
As track2kml is breaking compatibility with track2kml-ext, latent code within track2kml related with exporting to courageous format (`allow_to_courageous_option`) has been internally removed, simplifying the code. This change doesn't influence the program behaviour from the user's perspective, so should it appear in the changelog?

Reply:
As long as it doesn't modify the CLI itself (e.g. modifying options) or the KML output, it shouldn't be included in the CLI changelog